frontend/src/components/ViewTrips/trips-container.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Trip from './trip';
+
+import * as DATABASE from '../../constants/database.js';
+
+
+/**
+ * Returns a promise of a query object containg the array of Trip Documents
+ * corresponding to the trips that the current user is a collaborator on.
+ *
+ * @param {firebase.firestore.Firestore} db The Firestore database instance.
+ * @param {string} userEmail The email corresponding to the current user
+ *    logged in.
+ * @return {Promise<!firebase.firestore.QuerySnapshot>} Promise object
+ *    containing the query results with zero or more Trip  documents.
```

Comment:
Nit:
```suggestion
 *    containing the query results with zero or more Trip documents.
```

Reply:
Mentioned in Hiram's above and just committed change. Nice catch.